spec/expeditor/service_spec.rb

```diff
@@ -89,16 +89,18 @@
   end
 
   describe '#status' do
-    let(:service) { Expeditor::Service.new(sleep: 10) }
     it 'returns current status' do
+      # Set large value of period in case test takes long time.
+      service = Expeditor::Service.new(period: 100)
```

Comment:
> there are no advantage

I prefer `let` syntax to clarify the test target even if it isn't shared, but I don't have so strong opinion on this 🙆